terraform-infra-approvals/darts-shared-infrastructure.json

```diff
@@ -1,6 +1,13 @@
 {
   "resources": [
-    {"type": "azuread_application_password"}
+    {"type": "azuread_application_password"},
+    {"type": "azurerm_subnet"},
+    {"type": "azurerm_managed_disk"},
+    {"type": "azurerm_public_ip"},
```

Comment:
do you need a public IP? shouldn't you be connecting privately?

Reply:
should be resolved now.